src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxTests.cs

```diff
@@ -276,5 +279,54 @@ public void IsAttributeTargetSpecifier()
                 }
             }
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72300")]
+        public void TestAllKindsReturnedFromGetKindsMethodsExist()
+        {
+            foreach (var method in typeof(SyntaxFacts).GetMethods(BindingFlags.Public | BindingFlags.Static))
+            {
+                if (method.ReturnType == typeof(IEnumerable<SyntaxKind>) && method.GetParameters() is [])
+                {
+                    foreach (var kind in (IEnumerable<SyntaxKind>)method.Invoke(null, null))
+                    {
+                        Assert.True(Enum.IsDefined(typeof(SyntaxKind), kind), $"Nonexistent kind '{kind}' returned from method '{method.Name}'");
+                    }
+                }
+            }
+        }
+
+        [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/72300")]
+        [InlineData(nameof(SyntaxFacts.GetContextualKeywordKinds), SyntaxKind.YieldKeyword, SyntaxKind.FileKeyword)]
```

Comment:
>SyntaxKind.FileKeyword [](http://example.com/codeflow?start=92&length=22)

It feels like the test will require less maintenance if the end range value is exclusive, i.e. ```ElifKeyword``` for this range.

Reply:
This thread was marked as resolved, but I didn't see any response in the code or otherwise. Reactivating.